### ISSUE TITLE
Update 0714

### ISSUE THREAD

**Issue by TOSBase** [Treeofsavior/EnglishTranslation#497]
With the commit "Update 0714" which was supposed to change the key of some lines in the files, in ETC.tsv many older lines got reverted. Was this intended? Because it rather looks like an accident.

**Comment by imcgames** [comment]
More likely an accident.
Can you tell us some of the lines that got reverted so that we can check into it?

**Comment by TOSBase** [comment]
e.g.
ETC_20150317_000004 Humans sometimes hunt Large Kepa for food.
ETC_20150317_000069 No one has ever seen the face hidden behind the mask it wears.
ETC_20150317_000155 Dark Slime

Those lines got reverted in that commit but it's seems they already got added again in this commit: https://github.com/Treeofsavior/EnglishTranslation/commit/7f5a493214a00db8665452d372d3e2e4cd42c581 I haven't checked all lines but those that I checked seemed fine again so I gues this can be marked as solved.
